### PR TITLE
feat: `kaikasWallet` and `klaytn` chain

### PR DESCRIPTION
**Pull request by suropark**
## Changes  

- Added `kaikasWallet`

### CODE REVIEW

File: packages/rainbowkit/src/wallets/walletConnectors/kaikasWallet/kaikasWallet.svg
Comment (DanielSinclair) [inline packages/rainbowkit/src/wallets/walletConnectors/kaikasWallet/kaikasWallet.svg:1]: Please use as 28x28 SVG with an opaque background here

File: packages/rainbowkit/src/wallets/walletConnectors/kaikasWallet/kaikasWallet.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            description: 'wallet_connectors.kaikas.extension.step3.description',
+            step: 'refresh',
+            title: 'wallet_connectors.kaikas.extension.step3.title',
+          },
+        ],
+      },
+    },
+    createConnector: getInjectedConnector({
+      namespace: 'klaytn',
+    }),
```
Comment (DanielSinclair) [inline packages/rainbowkit/src/wallets/walletConnectors/kaikasWallet/kaikasWallet.ts:48]: Does this wallet support EIP-6963? This is the preferred standard going forward

**Review by DanielSinclair — CHANGES_REQUESTED** [review]

**Comment by magiziz** [comment]
@suropark Did a few tweaks. Added WalletConnect connector if injected wallet is not present + added a few more links. Realised also that Kaikas Wallet only works with klaytn chain so added that too 🙏

File: packages/rainbowkit/src/wallets/walletConnectors/kaikasWallet/kaikasWallet.svg
Comment (suropark) [reply packages/rainbowkit/src/wallets/walletConnectors/kaikasWallet/kaikasWallet.svg:1]: update to 28x28 svg  and white bg

File: packages/rainbowkit/src/wallets/walletConnectors/kaikasWallet/kaikasWallet.ts (hunk 1 above)
Comment (suropark) [reply packages/rainbowkit/src/wallets/walletConnectors/kaikasWallet/kaikasWallet.ts]: it does not support EIP-6963 yet.

**Comment by suropark** [comment]
> @suropark Did a few tweaks. Added WalletConnect connector if injected wallet is not present + added a few more links. Realised also that Kaikas Wallet only works with klaytn chain so added that too 🙏

appreciate it !

**Review by DanielSinclair — APPROVED** [review]

**Pull request merged**